libkbfs/journal_server.go

```diff
@@ -316,6 +316,11 @@ func (j *JournalServer) EnableExistingJournals(
 	eg, groupCtx := errgroup.WithContext(ctx)
 
 	fileCh := make(chan os.FileInfo, len(fileInfos))
+	type journalRet struct {
+		id      tlf.ID
+		journal *tlfJournal
+	}
+	journalCh := make(chan journalRet, len(fileInfos))
```

Comment:
Well it's the same size as `fileInfos` and `fileCh` themselves right?  So it doesn't add much more overhead compared to what's already there, as far as I can tell.  Plus, this is the same size as the eventual `tlfJournals`, which has to reside in memory.  So I don't think there's anything too scary about this.